frontend/src/libs/constants/constants.ts

```diff
@@ -1,2 +1,3 @@
+export { FIRST_INDEX, LAST_INDEX } from './index.constant.js';
 export { MOBILE_DIMENSION } from './media-query.constant.js';
 export { EMPTY_ARRAY_LENGTH, FIRST_ARRAY_INDEX } from 'shared/build/index.js';
```

Comment:
You already have `FIRST_ARRAY_INDEX` in shared constants, just add there constant for last index and use them

Reply:
done